packages/react-scripts/scripts/init.js

```diff
@@ -64,6 +101,10 @@ module.exports = function(appPath, appName, verbose, originalDirectory, template
     }
   });
 
+  if (gitInit()) {
```

Comment:
If we init git here, doesn't this mean adding `react` and `react-dom` dependencies are not added yet?